lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -883,6 +885,15 @@ SubExprInfo ExprEmitter::visitComb(ConcatOp op) {
   return {Unary, IsUnsigned};
 }
 
+SubExprInfo ExprEmitter::visitComb(BitcastOp op) {
+  Type toType = op.getType();
+  os << "/*cast(bit";
+  emitTypeDims(toType, op.getLoc(), os);
+  os << ")*/";
+  auto subPrec = emitSubExpr(op.from(), LowestPrecedence);
```

Comment:
Please add a big comment here saying that Bitcast is always emitted out of line with its own wire declaration to handle the cast.  Otherwise this looks very broken :-)